handler/dimensions.go

```diff
@@ -38,9 +111,10 @@ func (h *Dimensions) GetAll(w http.ResponseWriter, r *http.Request) {
 		PopulationType: chi.URLParam(r, "population-type"),
 	}
 	if err := parseRequest(r, &req); err != nil {
-		h.respond.Error(ctx, w, http.StatusBadRequest, &Error{
-			err: errors.Wrap(err, "invalid request"),
-		})
+		h.respond.Error(ctx, w,
```

Comment:
Rather than breaking the line arbitrarily it's better to format vertically
```
h.respond.Error(
  ctx,
  w,
  http.StatusBadRequest,
  errors.Wrap(err..),

Reply:
ah I see, I must have taken the &Error{} out of this one instead of my own method

Reply:
Okay that should be done now, ammended this, and the original one in my own method

Reply:
will leave you to resolve